### ISSUE TITLE
Can't seem to get it to compile on linux

### ISSUE THREAD

**Issue by drforbin** [SLASHEM-Extended/SLASHEM-Extended#47]
cc -g -O2 -Iinclude -DDLB -DDLBFILE=\"nhdat\" -DDLBAREA=FILE_AREA_SHARE -MMD -MP -c -o util/dgn_yacc.o util/dgn_yacc.c
dgn.tab.c:156:10: fatal error: dgn.tab.h: No such file or directory
compilation terminated.
make: *** [sys/unix/GNUmakefile:123: util/dgn_yacc.o] Error 1

HELP

**Comment by xade93** [comment]
I second that, bumped into same issue too.
moreover, seems like there is no file named `dgn.tab.h` after all.

**Comment by sidetrackinstar** [comment]
if theres no *.tab.h file then simply look for the directory that contains *.y and `bison -d *.y`
in this case (while in root of SLEX) `cd util && bison -d dgn_comp.y`

**Comment by 9sean6** [comment]
does it work after a (while in SLEX root) `cd util && bison -d dgn_comp.y && cp dgn_comp.tab.h dgn.tab.h` ?

**Comment by xade93** [comment]
Yes, it works. It seems like user need to manually bison "dgn_comp.y" and "lev_comp.y" in utils folder.

**Comment by Soviet5lo** [comment]
Seems like its due to a Bison change in v3.7 which released last year, so depending on distro used this may occur (I imagine just about any recent distro release since the beginning of the year includes 3.7 in its repos)
